tests/integration_tests/test_in_database_predicting_autopilot.py

```diff
@@ -1,17 +1,34 @@
 import pytest
-from tests.integration_tests.utils.parameters import aws_params, setup_params
+from tests.integration_tests.utils.parameters import aws_params, \
+    reg_setup_params, cls_setup_params
 
 
 @pytest.mark.skipif(not aws_params.aws_access_key,
                     reason="AWS credentials are not set")
 def test_predict_autopilot_udf(
-        register_language_container, deploy_scripts, setup_database):
-    db_conn = setup_database
-    prediction_udf_name = setup_params.endpoint_name
+         deploy_scripts, setup_database):
+
+    problem_types_dict = {
```

Comment:
better use parameterized tests here, or have two test which call the run_test with different parameter, because with for loop you run two tests in on test and if the first fails the second doesn't run